parallelization_test/parallelization-benchmark.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        results_df = pd.DataFrame(
</pre><p>would add the fetching of both total times here (see comment above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>